packages/react-integration/demo-app-ts/src/components/demos/GalleryDemo/GalleryDemo.tsx

```diff
@@ -5,7 +5,18 @@ export class GalleryDemo extends React.Component {
   static displayName = 'GalleryDemo';
   render() {
     return (
-      <Gallery hasGutter>
+      <Gallery
```

Comment:
I would add demo for each breakpoint.  We have instances in the past where certain breakpoints styles were not being applied as expected.

Reply:
I'm confused, by each breakpoint do you mean one demo for `minWidths` and one demo for `maxWidths` or do you mean one demo per size (sm/md/lg/etc) per property? The current test is testing the whole style object to ensure the classes are applied properly. Is there a different method of testing I should use in the cypress tests instead?

Reply:
@kmcfaul I would just add the missing breakpoints to what you have (xl and 2xl etc) for both the max and min widths to make sure they are applied correctly.

Reply:
Oh I understand, gotcha!